src/argilla/client/feedback/dataset/remote/base.py

```diff
@@ -102,7 +105,9 @@ def __len__(self) -> int:
         return response.parsed.records.count
 
     @allowed_for_roles(roles=[UserRole.owner, UserRole.admin])
-    def __getitem__(self, key: Union[slice, int]) -> Union[RemoteFeedbackRecord, List[RemoteFeedbackRecord]]:
+    def _get_records(
+        self, key: Union[slice, int], filters: Dict[str, Any] = {}
```

Comment:
If this is a base class, I'm not sure should have knowledge of `filters` argument, which is required only for one specific class implementation

Reply:
With base here I meant not base functionality but with the default methods, I can rename that to be a mixin separately for the requests to the API